tracardi/service/storage/drivers/elastic/tag.py

```diff
@@ -0,0 +1,47 @@
+from tracardi.service.storage.factory import storage_manager
+from typing import List
+
+
+async def get_document_by_event_type(event_type):
+    return await storage_manager("event-tags").load_by(field="type", value=event_type)
+
+
+async def add_tags(event_type: str, tags: List[str]):
+    search_result = await get_document_by_event_type(event_type=event_type)
+    if search_result.dict()["result"]:
```

Comment:
move search_result.dict() to variable. Making dict() form pydantic object can be expensive.